Kbs.Business/Boat/IBoatRepository.cs

```diff
@@ -7,5 +7,7 @@ public interface IBoatRepository
     List<BoatEntity> GetManyByName(string name);
     List<BoatEntity> GetManyByNameAndType(string name, int boatTypeId);
     BoatEntity GetById(int boatId);
+    void UpdateDeleteRequestDate(int boatId, DateTime? requestDate, BoatStatus status);
```

Comment:
Vervang met `Update(BoatEntity boat)`, deze update gewoon alle properties van een boot i.p.v. 1 of 2:
```suggestion
    void Update(BoatEntity boat);
```